creator-node/src/routes/userReplicaSet.js

```diff
@@ -0,0 +1,203 @@
+const axios = require("axios")
+const { promisify } = require('util')
+const crypto = require('crypto')
+const randomBytes = promisify(crypto.randomBytes)
+const { Utils: LibsUtils } = require('@audius/libs')
+
+const {
+  handleResponse,
+  errorResponseBadRequest,
+  successResponse,
+  parseCNodeResponse,
+  errorResponseServerError
+} = require("../apiHelpers")
+const { logger } = require('../logging')
+const config = require('../config')
+const { generateTimestampAndSignature, recoverWallet, signatureHasExpired } = require("../apiSigning")
+
+module.exports = function (app) {
+  /**
+   * TODO
+   * - rate limit
+   * - consider also validating L1 healthy state eg valid bounds
+   * - move to component pattern
```

Comment:
component pattern should get done before we merge this i think - let's not introduce tech debt intentionally if avoidable

Reply:
this is done, in new pr